### PR TITLE
#1 Add Edit Dialog, Add summary dialog

### PR DESCRIPTION
**Pull request by HidekiNakazawa**

### CODE REVIEW

File: src/js/bootstrap-wysiwyg.js [hunk 1]
```diff
@@ -0,0 +1,200 @@
+/* http://github.com/mindmup/bootstrap-wysiwyg */
```
Comment (dixonsiu) [inline src/js/bootstrap-wysiwyg.js]: Please use the latest version of bootstrap-wysiwyg with script tag.
You can find the latest version [here](https://cdnjs.com/libraries/bootstrap3-wysiwyg).

**Review by dixonsiu — CHANGES_REQUESTED** [review]

File: src/index.html [hunk 2]
```diff
@@ -20,26 +20,32 @@
   <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery-i18next/1.2.0/jquery-i18next.min.js"></script>
   <script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.3.7/js/bootstrap.min.js"></script>
 
+  <script src="./js/bootstrap-wysiwyg.js"></script>
+
 <script type="text/javascript">
 
-// 現在表示中の機能ID
+// Function ID currently displayed
```
Comment (dixonsiu) [inline src/index.html]: Please move all JavaScript into JS file and make sure it will work when loaded asynchronously.

**Review by dixonsiu — CHANGES_REQUESTED** [review]

File: src/index.html (hunk 2 above)
Comment (atsX) [reply src/index.html]: resolved : pull request #10

**Review by atsX — COMMENTED** [review]
change to use FontAwesome from img

**Review by dixonsiu — APPROVED** [review]

**Comment by atsX** [comment]
Merge to take over to another worker

**Pull request merged**